test/cppsuite/test_harness/workload_generator.h

```diff
@@ -67,38 +98,37 @@ class workload_generator : public component {
     void
     run() override final
     {
-        configuration *read_config, *update_config, *insert_config;
+        configuration *populate_config;
+        std::vector<operation_config> operation_configs;
+        uint64_t thread_id = 0;
+
+        /* Retrieve useful parameters from the test configuration. */
+        operation_configs.push_back(
```

Comment:
We could iterate the enum here but enum's aren't meant to be iterated so I think we can leave it as is for now.